### PR TITLE
Feature/arc 1246

### PR DESCRIPTION
**Pull request by silkederudder-shd**
image

### CODE REVIEW

**Review by WardVerc1 — APPROVED** [review]

File: public/locales/nl/common.json [hunk 1]
```diff
@@ -200,12 +200,12 @@
   "modules/media/const/index___dit-object-bevat-geen-licentie-voor-online-raadpleging-breng-een-bezoek-aan-de-bezoekersruimte": "Dit object bevat geen licentie voor online raadpleging. Contacteer de aanbieder voor meer informatie.",
   "modules/media/const/index___duurtijd": "Duurtijd",
   "modules/media/const/index___episode": "Aflevering",
-  "modules/media/const/index___er-ging-iets-mis-bij-het-ophalen-van-het-afspeel-token": "Er ging iets mis bij het afspelen. Probeer het opnieuw.",
+  "modules/media/const/index___er-ging-iets-mis-bij-het-ophalen-van-het-afspeel-token": "Er ging iets mis bij het afspelen. Probeer het opnieuw. ",
```
Comment (bertyhell) [inline public/locales/nl/common.json:203]: remove trailing space

File: public/locales/nl/common.json [hunk 2]
```diff
@@ -491,14 +491,15 @@
   "pages/admin/gebruikersbeheer/permissies/index___opnieuw-instellen": "Opnieuw instellen",
   "pages/admin/gebruikersbeheer/permissies/index___uitvoeren": "Uitvoeren",
   "pages/admin/gebruikersbeheer/permissies/index___wijzigingen-opslaan": "Wijzigingen opslaan",
+  "pages/admin/mijn-materiaalaanvragen/index___mijn-materiaalaanvragen": "",
+  "pages/admin/mijn-materiaalaanvragen/index___mijn-materiaalaanvragen-meta-omschrijving": "",
```
Comment (bertyhell) [inline public/locales/nl/common.json:495]: Add a default translation in the common.json file for these 2 keys

**Review by bertyhell — CHANGES_REQUESTED** [review]

File: public/locales/nl/common.json [hunk 3]
```diff
@@ -42,6 +42,7 @@
   "modules/admin/const/routing___gebruikers": "Gebruikers",
   "modules/admin/const/routing___gebruikersbeheer": "Gebruikersbeheer",
   "modules/admin/const/routing___groepen-en-permissies": "Groepen en permissies",
+  "modules/admin/const/routing___mijn-materiaalaanvragen": "",
```
Comment (bertyhell) [inline public/locales/nl/common.json:45]: add default string for all translations with "" as value

**Comment by bertyhell** [comment]
this translation still contains a trailing slash:
```
  "modules/media/const/index___er-ging-iets-mis-bij-het-ophalen-van-het-afspeel-token": "Er ging iets mis bij het afspelen. Probeer het opnieuw. ",
```

File: src/pages/admin/materiaalaanvragen/index.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+		<>
+			{renderOgTags(
+				tText('pages/admin/mijn-materiaalaanvragen/index___mijn-materiaalaanvragen'),
+				tText(
+					'pages/admin/mijn-materiaalaanvragen/index___mijn-materiaalaanvragen-meta-omschrijving'
+				),
+				url
+			)}
+
+			<PermissionsCheck allPermissions={[]}>{renderPageContent()}</PermissionsCheck>
```
Comment (bertyhell) [inline src/pages/admin/materiaalaanvragen/index.tsx:37]: You can create a permission and give it the same name as the backend:
https://github.com/viaacode/hetarchief-proxy/blob/release/archief2-fase2/src/modules/users/types.ts#L53

you can add the permissions here:
https://github.com/viaacode/hetarchief-client/blob/master/src/modules/account/const/index.tsx#L90

The 2 Permission enum objects in back and frontend should stay in sync at all time

**Review by bertyhell — APPROVED** [review]

**Pull request merged**